test/e2e/suite/command/tag.go

```diff
@@ -75,51 +75,29 @@ var _ = Describe("OCI image layout users:", func() {
 		ORAS("repo", "tags", Flags.Layout, LayoutRef(root, "")).MatchKeyWords(tags...).Exec()
 	}
 	When("running `tag`", func() {
-		prepare := func() string {
-			root := GinkgoT().TempDir()
-			ORAS("cp", RegistryRef(Host, ImageRepo, multi_arch.Tag), Flags.ToLayout, LayoutRef(root, multi_arch.Tag)).Exec()
-			return root
-		}
 		It("should add a tag to an existent manifest when providing tag reference", func() {
-			tagAndValidate(prepare(), multi_arch.Tag, multi_arch.Digest, "tag-via-tag")
+			tagAndValidate(PrepareTempOCI(ImageRepo), multi_arch.Tag, multi_arch.Digest, "tag-via-tag")
 		})
 		It("should add a tag to an existent manifest when providing digest reference", func() {
-			tagAndValidate(prepare(), multi_arch.Digest, multi_arch.Digest, "tag-via-digest")
+			tagAndValidate(PrepareTempOCI(ImageRepo), multi_arch.Digest, multi_arch.Digest, "tag-via-digest")
 		})
 		It("should add multiple tags to an existent manifest when providing digest reference", func() {
-			tagAndValidate(prepare(), multi_arch.Digest, multi_arch.Digest, "tag1-via-digest", "tag2-via-digest", "tag3-via-digest")
+			tagAndValidate(PrepareTempOCI(ImageRepo), multi_arch.Digest, multi_arch.Digest, "tag1-via-digest", "tag2-via-digest", "tag3-via-digest")
 		})
 		It("should add multiple tags to an existent manifest when providing tag reference", func() {
-			tagAndValidate(prepare(), multi_arch.Tag, multi_arch.Digest, "tag1-via-tag", "tag1-via-tag", "tag1-via-tag")
+			tagAndValidate(PrepareTempOCI(ImageRepo), multi_arch.Tag, multi_arch.Digest, "tag1-via-tag", "tag1-via-tag", "tag1-via-tag")
 		})
-	})
-})
-
-var _ = Describe("OCI image layout users:", func() {
-	var tagAndValidate = func(root string, tagOrDigest string, digest string, tags ...string) {
-		out := ORAS(append([]string{"tag", LayoutRef(root, tagOrDigest), Flags.Layout}, tags...)...).MatchKeyWords(tags...).Exec().Out
-		hint := regexp.QuoteMeta(fmt.Sprintf("Tagging [oci-layout] %s", LayoutRef(root, digest)))
-		gomega.Expect(out).To(gbytes.Say(hint))
-		gomega.Expect(out).NotTo(gbytes.Say(hint)) // should only say hint once
-		ORAS("repo", "tags", Flags.Layout, LayoutRef(root, "")).MatchKeyWords(tags...).Exec()
-	}
-	When("running `tag`", func() {
-		prepare := func() string {
-			root := GinkgoT().TempDir()
-			ORAS("cp", RegistryRef(Host, ImageRepo, multi_arch.Tag), Flags.ToLayout, LayoutRef(root, multi_arch.Tag)).Exec()
-			return root
```

Comment:
Do we not need this?

Reply:
because we are using ZOT and can copy from oci layout directly

Reply:
How about this part?

```go
ORAS("repo", "tags", Flags.Layout, LayoutRef(root, "")).MatchKeyWords(tags...).Exec()
```

Reply:
see https://github.com/oras-project/oras/blob/main/test/e2e/internal/utils/prepare.go#L41

Reply:
OK this part is not actually removed. The git diff rendering is kinda misleading.